entity-framework/core/miscellaneous/cli/dotnet.md

```diff
@@ -206,6 +206,18 @@ The following example scaffolds only selected tables and creates the context in
 dotnet ef dbcontext scaffold "Server=(localdb)\mssqllocaldb;Database=Blogging;Trusted_Connection=True;" Microsoft.EntityFrameworkCore.SqlServer -o Models -t Blog -t Post --context-dir Context -c BlogContext --context-namespace New.Namespace
 ```
 
+The following example demonstrates using the `name=<name of connection string>` syntax to get the connection string from the configuration sources that are set up for the project. In this case "BloggingConnection" would be the name of a connection string in configuration.
+
+```dotnetcli
+dotnet ef dbcontext scaffold "name=BloggingConnection" Microsoft.EntityFrameworkCore.SqlServer
+```
+
+The following example skips scaffolding an `OnConfiguring` method. This can be useful for ensuring a development connection string or one with a username and password does not get written into the DbContext that is scaffolded. Added in EF Core 5.0.
```

Comment:
```suggestion
The following example skips scaffolding an `OnConfiguring` method. This can be useful when you want to configure the DbContext outside of the class. For example, ASP.NET Core apps typically configure it in Startup.ConfigureServices. Added in EF Core 5.0.
```